### PR TITLE
Add style

### PR DESCRIPTION
**Pull request by zamcham**
This update adds the styling meeting the following requirements:

1. The final design should match Zeplin's design (this includes all elements + the same styling)
2. Used the CSS rules detailed in Zeplin to style the website.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,4 +1,317 @@
...
+}
+
+.navBar {
+  display: flex;
+  margin: auto;
+  padding: 1rem 6rem;
+  justify-content: space-between;
+  background-color: white;
+  border-bottom: solid 1px #e8e8e8;
+}
```
Comment (franclobo) [inline src/App.css]: - [x] Please 🙏🏽 add te login icon to b similar to the Zeplin design,

| Zeplin | Your design |
|---|---|
| image | image |

File: src/App.css [hunk 2]
```diff
@@ -1,4 +1,317 @@
...
+  justify-content: space-between;
+  align-items: center;
+  width: 75rem;
+  height: 10.625rem;
+  margin: 1rem auto;
+  padding: 2rem 9.188rem 1.625rem 1.688rem;
+  border-radius: 4px;
+  background-color: #fff;
+  border: 1px solid #e8e8e8;
+}
```
Comment (franclobo) [inline src/App.css]: - [x] Kindly align the book panel with the title to be similar to the Zeplin design.

| Zeplin | Your design |
|---|---|
| image | image |

File: src/components/AddBookForm.jsx [hunk 3]
```diff
@@ -26,7 +26,7 @@ function AddBookForm() {
   };
 
   return (
-    <form onSubmit={handleSubmit}>
+    <form className='addForm' onSubmit={handleSubmit}>
```
Comment (franclobo) [inline src/components/AddBookForm.jsx]: - [x] Please 🙏🏽 add the title "ADD NEW BOOK" to be similar to the Zeplin design.

| Zeplin | Your design |
|---|---|
| image | image |

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @zamcham, 

# ♻️ Required Changes ♻️

so-close-jim-halpert

Good job so far!
You are doing it well 👏🏽 there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- No linter errors ✔️ 
- Descriptive PR ✔️ 
- Correct Git flow ✔️ 
- Good styles ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by topeogunleye — APPROVED** [review]
### Hi @zamcham, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- Made UI changes as suggested ✔️
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
 
## [Optional] suggestions:
* [Nothing to mention.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**